lib/dynamo.js

```diff
@@ -22,8 +22,20 @@ function createClient(config) {
   }
 
   const client = new AWS.DynamoDB.DocumentClient(config);
+  client.consistentReads = 'consistentReads' in config && config.consistentReads;
```

Comment:
say what now? what are you trying to do here? make sure it's not undefined? you can use `_.get(config, 'consistentReads', false)`

Reply:
sure, good call

Reply:
or just config.consistenReads || false since i doubt config will ever be undefined